server/scripts/populate.py

```diff
@@ -0,0 +1,150 @@
+import psycopg2
+import yaml  # noqa
```

Comment:
I use Anaconda, and it was complaining about not being able to find yaml. It ran perfectly well, so I silenced it.

Reply:
According to conda issue 5142 (which I will not link here to avoid showing up in their issue), you might have to `conda install` instead of `pip install`

Reply:
Fixed; vscode issue.